src/pages/user/authority/ConnectAuthority.tsx

```diff
@@ -40,16 +40,26 @@ export const ConnectAuthority: React.FC = () => {
     }
   }, [user.possibleAuthorities]);
 
-  const setOrcidAndFeide = async () => {
+  const updateAuthorityForUser = async () => {
     const selectedAuthority = matchingAuthorities.find(
       auth => auth.systemControlNumber === selectedSystemControlNumber
     );
 
     if (selectedAuthority && user.authority) {
       selectedAuthority.orcids.length > 0 && dispatch(setOrcid(selectedAuthority.orcids));
 
-      const updatedAuthority = await updateFeideForAuthority(user.id, user.authority.systemControlNumber, dispatch);
-      dispatch(setAuthorityData(updatedAuthority));
+      const updatedAuthorityWithFeide = await updateFeideForAuthority(
+        user.id,
+        user.authority.systemControlNumber,
+        dispatch
+      );
+      if (updatedAuthorityWithFeide) {
+        const updatedAuthorityWithOrganizationId = await updateInstitutionForAuthority(
+          user.organizationId,
+          user.authority.systemControlNumber
+        );
+        dispatch(setAuthorityData(updatedAuthorityWithOrganizationId));
+      }
```

Comment:
dette er endret på nå, så min PR burde kanskje inn først (men jeg er borte i morgen, så kanskje lettere å copy-paste fra min PR isåfall)